### PR TITLE
icon of app-extension is fixed

### PR DESCRIPTION
**Pull request by shivam2562001**
I have fixed  icon issue in chrome extensions.

Issue Tag: white dev backpack icon not visible on default windows chrome theme #3211

### CODE REVIEW

**Comment by wentokay** [comment]
hey @shivam2562001 this won't work unfortunately as [webpack will copy](https://github.com/coral-xyz/backpack/blob/64b3d08ed07519f403b0cf63e1a475b1a8222d81/packages/app-extension/webpack.config.js#L266-L271) `anchor-production.png` or `anchor-development.png` to `anchor.png` depending on whether or not it is in the production (yarn build) or development (yarn start) environment.

We currently use a red and white icon to differentiate the production and development build icons respectively.

What we really need here is to have an icon that will still be visible on the default white browser toolbar in windows. I think what we might want is black outline or perhaps a different colored icon so that it's still visible.

@trucy @aarongillett do you have any preference for the development build icon color related to #3211? I thought adding an outline might be useful in case a users' browser is the exact same color as the icon, right now we're assuming that nobody is using a bright red browser theme for instance.

**Comment by aarongillett** [comment]
Maybe just add a grey or red outline for now (if you haven’t already). We can revisit with more consideration and proper cross-browser testing later.

File: yarn.lock [hunk 1]
```diff
@@ -19987,10 +19987,10 @@ __metadata:
 
 "eslint-plugin-zeus-custom@file:./eslint-plugin-zeus-custom::locator=eslint-config-custom%40workspace%3Apackages%2Feslint-config-custom":
   version: 0.0.1
-  resolution: "eslint-plugin-zeus-custom@file:./eslint-plugin-zeus-custom#./eslint-plugin-zeus-custom::hash=a463b5&locator=eslint-config-custom%40workspace%3Apackages%2Feslint-config-custom"
```
Comment (hkirat) [inline yarn.lock:19990]: Can u revert this?

Comment (shivam2562001) [reply yarn.lock]: reverted

**Comment by shivam2562001** [comment]
I have added black color icon for anchor-development


anchor-development

**Comment by shivam2562001** [comment]
image

**Comment by trucy** [comment]
@shivam2562001 here's a version of the logo with red outline and white background. 

@johnrees I can make any color versions you might need (black outline or red outline transparent fill etc) just lmk!

backpack logo (white with red outline)

**Comment by shivam2562001** [comment]
yeah red outline looks cool I will add this icon in manifest

**Comment by shivam2562001** [comment]
I have added above outline icon. I think this fits with the requirement.

ss

can someone pls approve this pr

**Review by aarongillett — COMMENTED** [review]
This icon is horizontally stretched. Let’s use the icon Trucy provided [here](https://github.com/coral-xyz/backpack/pull/3468#issuecomment-1494782731).

**Comment by shivam2562001** [comment]
I used the same icon with previous dimension of 72x72

Let me know should I use the same dimension as trucy provided?

**Pull request closed without merging**